atr/storage/writers/distributions.py

```diff
@@ -23,6 +23,7 @@
 
 import aiohttp
 import sqlalchemy.exc as exc
+from sqlalchemy import text
```

Comment:
We don't use this import style, [according to our code conventions](https://release-test.apache.org/docs/code-conventions).

Reply:
The [PEP8 convention for denoting private interfaces](https://peps.python.org/pep-0008/#public-and-internal-interfaces) is to use a single leading underscore in the interface name. We can use any interface of any dependency which does not begin with a single underscore.

We do not rename interfaces when we import them, as our code conventions state. Our standard form is `import a.b as b`, but if importing as `b` causes problems the best alternative is usually `import a.b`.